Zend/zend_exceptions.c

```diff
@@ -639,11 +643,13 @@ ZEND_METHOD(Exception, getTraceAsString)
 /* {{{ Return previous Throwable or NULL. */
 ZEND_METHOD(Exception, getPrevious)
 {
-	zval rv;
+	zval *prop, rv;
 
 	ZEND_PARSE_PARAMETERS_NONE();
 
-	ZVAL_COPY(return_value, GET_PROPERTY_SILENT(ZEND_THIS, ZEND_STR_PREVIOUS));
+	prop = GET_PROPERTY(ZEND_THIS, ZEND_STR_PREVIOUS);
```

Comment:
Is the change from non-silent to silent intentional?

Reply:
Nice catch!